python/setup.py

```diff
@@ -95,7 +95,7 @@ def is_pure(self):
         'decorator',
         ],
       packages=find_packages(),
-      include_package_data=True,
+      include_package_data=False,
```

Comment:
@tqchen Could you please check the setup.py? If this line is True, TVM could not be installed after the it is built, similar to this issue: https://github.com/dmlc/xgboost/issues/1846. Thank you.